apps/context/src/parser/python.ts

```
@@ -0,0 +1,379 @@
+/**
+ * Python tree-sitter parser.
+ *
+ * Extracts symbols (functions, classes, methods) and their metadata
+ * from Python source files using tree-sitter.
+ */
+
+import Parser from "tree-sitter";
+// eslint-disable-next-line @typescript-eslint/no-require-imports
+const PythonLang = require("tree-sitter-python") as Parser.Language;
+import type { ParsedFile, Symbol } from "../types.js";
```

Biome (2.4.6)

[error] Do not shadow the global "Symbol" property.

(lint/suspicious/noShadowRestrictedNames)